etc/Dictionaries/DFLError_en_GB.dic

```diff
@@ -9,8 +9,11 @@
 
 //------------------ Inputs -------------------------
 ErrorConfigFileRead           =     error while reading configuration file: %1%
+ChosenOutputDoesntExist       =     %1% chosen output doesn't exist
 ContingenciesReadError        =     error while reading contingencies file %1%: %2%
 DynModelFileReadError         =     cannot parse %1% : %2%
+SimulationKindDoesntExist     =     %1% simulation kind doesn't exist
+EmptyMap                      =     map is empty
```

Comment:
Error message should not be created for tests, please simply crash with a ASSERT_FALSE(map.empty())